packages/frontend/core/src/components/affine/setting-modal/general-setting/plans/index.tsx

```diff
@@ -175,18 +204,19 @@ const Settings = () => {
             </RadioButton>
           ))}
         </RadioButtonGroup>
-        {/* TODO: plan cards horizontal scroll behavior is not the same as design */}
-        {/* TODO: may scroll current plan into view when first loading? */}
-        <div className={styles.planCardsWrapper}>
+        <div className={styles.planCardsWrapper} ref={scrollWrapper}>
           {Array.from(planDetail.values()).map(detail => {
+            const isCurrent =
+              loggedIn &&
+              detail.plan === currentPlan &&
+              (currentPlan === SubscriptionPlan.Free
+                ? true
+                : currentRecurring === recurring);
             return (
```

Comment:
Need a PriceCard component to refactor those codes, and style is incorrect now. 
![image](https://github.com/toeverything/AFFiNE/assets/9527419/ddc031f6-1891-4271-b4d7-2f7af85f5137)